common/biovotion/biovotion_vsm_battery_state.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.biovotion",
+  "type": "record",
+  "name": "BiovotionVSMBatteryState",
+  "doc": "Device battery state.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "batteryLevel", "type": "float", "doc": "battery level (0-1)"},
+    {"name": "batteryChargeRate", "type": "float", "doc": "battery charge rate (0-1)"},
+    {"name": "batteryVoltage", "type": "float", "doc": "battery voltage (V)"},
+    {"name": "batteryStatus", "type": "float", "doc": "battery status"}
```

Comment:
Would it be possible to use an enum here? In server status for example, on record is:
```json
    {"name": "serverStatus", "type": {"name": "ServerStatus", "type": "enum", "symbols": ["CONNECTED", "DISCONNECTED", "UNKNOWN"]}, "doc": "server connection status", "default": "UNKNOWN"},
```
Otherwise, perhaps a reference to what the state is supposed to mean?